babel.config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Not sure what this is for - I tried running `make build` both before and after deleting it and the outcome looked the same.

Reply:
That might just be because nx has cached it: if you run the following it should throw an error with the babel config:
```
pnpm nx run-many --target=build --all=true --skip-nx-cache

```

Reply:
The error is: 
```
 Error during bundle: Error: Babel was run with rootMode:"upward" but a root could not be found when searching upward from "/Users/Joe_Cowton/code/csnx/libs/tsconfig".
 ```
 
 I linked the issue in the original PR comment but for reference it's here: https://github.com/nrwl/nx/issues/3184#issuecomment-674770770

Reply:
Ah yes, got that error too. It was solved when I changed the compiler to swc.